docs/reference/commandline/create.md

```diff
@@ -42,6 +42,7 @@ Creates a new container.
       --help=false                  Print usage
       -i, --interactive=false       Keep STDIN open even if not attached
       --ipc=""                      IPC namespace to use
+      --isolation=                  Container isolation level
```

Comment:
Same question here: do yo think the flag description should be updated at the same time? Because they are the same.